packages/collector/test/tracing/cloud/aws-sdk/v3/sns/test_definition.js

```diff
@@ -0,0 +1,305 @@
+/*
+ * (c) Copyright IBM Corp. 2023
+ */
+
+'use strict';
+
+const semver = require('semver');
+const { expect } = require('chai');
+const { fail } = expect;
+const qs = require('querystring');
+const path = require('path');
+const supportedVersion = require('@instana/core').tracing.supportedVersion;
+const config = require('../../../../../../../core/test/config');
+const { retry, delay, stringifyItems, expectExactlyOneMatching } = require('../../../../../../../core/test/test_util');
+const ProcessControls = require('../../../../../test_util/ProcessControls');
+const globalAgent = require('../../../../../globalAgent');
+const {
+  verifyHttpRootEntry,
+  verifyExitSpan,
+  verifyHttpExit
+} = require('@instana/core/test/test_util/common_verifications');
+const constants = require('@instana/core').tracing.constants;
+const utils = require('./utils');
+
+let topicArn;
+const topicName = 'MyNodeTopicArn';
+const availableStyles = ['default', 'callback', 'v2'];
+const availableCommands = {
+  PublishCommand: {
+    Message: 'STRING_VALUE',
+    TargetArn: 'STRING_VALUE',
+    Subject: 'STRING_VALUE'
+  }
+};
+
+/*
+fs
+  .readdirSync(`${path.dirname(require.resolve('@aws-sdk/client-sns'))}/commands`)
```

Comment:
Suggestion: Can we remove this commented code if it is no longer needed?

Reply:
Yeah sure we can do. I wasn't sure tbh